spec/patterns/core_spec.rb

```diff
@@ -108,10 +108,99 @@
 
   context "when using comma separators and other regexp" do
 
+    let(:pattern) { '((a=(?<a>%{UNIXPATH})?|b=(?<b>%{UNIXPATH})?)(,\s)?)+' }
+
+    let(:grok) do
+      grok = LogStash::Filters::Grok.new("match" => ["message", pattern])
+      grok.register
+      grok
+    end
+
     let(:value) { 'a=/some/path, b=/some/other/path' }
 
+    it "should match both paths" do # but does not
+      event = build_event(value)
+      grok.filter(event)
+      expect( event.to_hash['a'] ).to eql '/some/path,'
```

Comment:
this case is an attempt to reproduce https://github.com/logstash-plugins/logstash-patterns-core/issues/13 which was supposed to be fixed using the [UNIXPATH](https://github.com/logstash-plugins/logstash-patterns-core/commit/06de0c1bc566874ff87d9b23765f4fee1b3a573f) (changed here) 
turns out these case weren't being handled (previous spec attempt gave a false impression) so the behaviour did not change in this PR.